### PR TITLE
FIX mrp: wrong update _set_date_deadline()

### PR DESCRIPTION
**Pull request by bealdav**
## Description of the issue/feature this PR addresses:

compute the right value for each record

## Current behavior before PR:

compute the same value for all records

## Desired behavior after PR is merged:

compute the right value for each record

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@sle-odoo for you!

**Review by legalsylvain — APPROVED** [review]

**Review by Yenthe666 — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Comment by ryv-odoo** [comment]
Hello, thank you for the PR.
It shouldn't target the saas-13.5?

cc @amoyaux

**Comment by bealdav** [comment]
@ryv-odoo I may update the target

**Review by OpenCode — APPROVED** [review]

**Review by astirpe — APPROVED** [review]

File: addons/mrp/models/mrp_production.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ def _compute_date_deadline(self):
 
     def _set_date_deadline(self):
         for production in self:
-            (self.move_raw_ids | self.move_finished_ids).date_deadline = production.date_deadline
+            (production.move_raw_ids | production.move_finished_ids).date_deadline = production.date_deadline
```
Comment (fpodoo) [inline addons/mrp/models/mrp_production.py:318]: As there is propagation of date changes amongst linked moves, I believe it's more accurate to do this instead:
    production.move_finished_ids.date_deadline = production.date_deadline

The date on raw materials will be triggered after and recomputed preserving the manufacturing lead time.

**Review by mourad-ehm — APPROVED** [review]

**Comment by ryv-odoo** [comment]
Fixed in 876d575e2ddacf504513d63299c6713178109ae6

**Pull request closed without merging**